src/roadmap/_roadmap.scss

```diff
@@ -56,7 +62,11 @@
   &__item {
     display: flex;
     justify-content: space-between;
-    width: 100%;
+    margin: 8px 24px;
+
+    &:last-of-type {
+      margin-bottom: 0px;
```

Comment:
Linters usually complain about add and units to a `0` value.

Consider this instead:
```
margin-bottom: 0;
```

Reply:
Woops! Updated